drivers/audio/Kconfig.st_dmic_i2s

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2018 STMicroelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig AUDIO_ST_DMIC_I2S
```

Comment:
And is that compliant with https://github.com/zephyrproject-rtos/zephyr/pull/11854?

Reply:
Well, this driver requires I2S name because does not depend to what ST microphone we use, but how we access it. In this case we use I2S. Same microphone can be attached through different communication and will require a different driver.

Before the driver name was the microphone one (i.e. mp34dt05.c) but I changed it because it does not work if same microphone is attache to other bus. For example the disco_l475_iot1 board has same microphone but we cannot use this driver.

Reply:
ok, clear enough